joomla/joomla-cms
# [4.0] Cleanup privacy menu

### Summary of Changes
Moves the submenu into the main left sidebar:
<img width="244" alt="image" src="https://user-images.githubusercontent.com/359377/57200995-f3efd400-6f92-11e9-9f00-6890a9a4d49d.png">

The layouts are updated using the namespaces.

### Testing Instructions
1. Login to the backend
2. Click on Privacy under Users
3. Notice the dashboard opens with the submenu in it.
4. Click the submenu items, the same submenu is visible on these pages
5. Apply pull request
6. Click Privacy under Users
7. A submenu folds open with the Privacy views
8. Click each view to see if it works

### Expected result
Menu items below Privacy

### Actual result
Menu items on the view page

### Documentation Changes Required
None

@wilsonge
